### ISSUE TITLE
Solr Search Should be On

### ISSUE THREAD

**Issue by dmlb2000** [pacifica/pacifica-userinterface-core#5]
Solr search is currently not on by default. The module seems to be available by some dependency in the profile but it's not enabled or configured. It should be.

**Comment by dmlb2000** [comment]
Issues with Acquia:

Lightning comes with Solr search but it's a dependency of their Acquia Connector plugin. That has a locked version of 1.2 which is not the current version of the Solr plugin (3.x). The 1.2 version of the plugin seems to only support up to Solr 6. The 3.x version of the Solr plugin supports Solr 7 and 8. It seems these are very big updates.

I would put in an issue but that's all tracked with Acquia proper and I can't submit an issue without some sort of poke into Acquia to let me.

**Comment by BenjaminStuermer** [comment]
As I understand it, questions of what Lighting supports or doesn't are out of scope for pacifica-userinterface-core, which defines a module rather than a full site with an install profile. If there's still a question of basing our install profile on Lighting that would be an issue to discuss in the scope of the project that defines our site-wide install profile, but since (as I understand it) Solr is really a hard dependency of the search capability, which is defined in userinterface-core, I think we would be installing and configuring the Solr module as a dependency of this module. I'd assume we'd be making that dependency to the most up-to-date version of Solr available, unless there's something problematic about the more recent releases.

@wats958 and @jamesmarks, does that seem right to you?

**Comment by wats958** [comment]
Yes, if we are making our own install profile eventually we do not want to make another profile a dependency (or expect it and its dependencies to be included). The module should be built to encompass all of its own needs.

The appeal of lightning is to 1) get a headstart esp. if you don't know what you're doing at first or what dependencies to start with in site building, and 2) to prepare your project to integrate with Acquia. The first isn't the case and the second isn't an expectation either for us or future Pacifica users.

It may be good to set up the submodules now as James described, so that solr can be added as a dependency of the "pacifica_search" submodule.
